drivers/eaton-mib.c

```diff
@@ -237,10 +245,111 @@ static info_lkp_t outlet_switchability_info[] = {
 	{ 0, NULL }
 };
 
+static info_lkp_t marlin_ambient_presence_info[] = {
+	{ -1, "unknown" },
+	{ 0, "no" },  /* disconnected */
+	{ 1, "yes" }, /* connected */
+	{ 0, NULL }
+};
+
+static info_lkp_t marlin_threshold_status_info[] = {
+	{ 0, "good" },          /* No threshold trigged */
+	{ 1, "warning-low" },   /* Warning low threshold trigged */
+	{ 2, "cricital-low" },  /* Critical low threshold trigged */
```

Comment:
"criCiTal"? Where is the typo - here in code or in some external counteragent which we must match?